analyzer/codechecker_analyzer/analyzers/clangtidy/analyzer.py

```diff
@@ -227,16 +229,30 @@ def get_checker_list(self, config) -> Tuple[List[str], List[str]]:
             state, _ = value
 
             # Checker name is a compiler warning.
+            if checker_name.startswith('W'):
+                LOG.warning(f"The following usage of {checker_name} "
+                            "compiler warning as a checker name is "
+                            "deprecated, please use "
+                            "clang-diagnostic-<checker-name> instead")
```

Comment:
This feels a little sterile. How about "Since 6.22.0, using 'W' prefix for compiler warnings 
 is deprecated. Please change from '`{checker_name}`' to 'clang-diagnostic-`{checker_name[1:]}'`."

Reply:
Good suggestion, thanks.